src/model/documentselection.js

```diff
@@ -237,7 +237,8 @@ export default class DocumentSelection {
 	 *
 	 * This method's result can be used for example to apply block styling to all blocks covered by this selection.
 	 *
-	 * **Note:** `getSelectedBlocks()` always returns the deepest block.
+	 * **Note:** `getSelectedBlocks()` returns blocks that are nested in other non-block elements
+	 * but will not return blocks nested in other blocks.
 	 *
 	 * In this case the function will return exactly all 3 paragraphs:
```

Comment:
Could you please change `quote` in the below example to `blockQuote`? AFAIR the element in the model is called`blockQuote`.